jeancochrane/open-apparel-registry
# [TESTING] Test django-ecsmanage package

## Overview

Temporarily remove the local implementation of ecsmanage.py and pin an installation of django-ecsmanage to the GitHub package in order to test the remote package.

Connects https://github.com/azavea/operations/issues/296

## Testing Instructions

* Shell into the VM
* Run `./scripts/manage ecsmanage --env staging help`, the equivalent of `python manage.py help`
  
  * Note that this will run on the staging environment by default
* Click on the link to the details page and confirm that it takes you to the right place
* On the details page, navigate to `Containers > django > Log Configuration`, open up the logs, and confirm that the command exited successfully
